infra.bs

```diff
@@ -102,6 +102,17 @@ retrieving and rendering web content and facilitating end user interaction with
 specifications using the Infra Standard, the user agent is generally the client software that
 implements the specification.
 
```

Comment:
Maybe add a `<hr>` before the definition of user agent now, to group the two together.